app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -744,16 +745,16 @@ class BrowserTabFragment :
         externalCameraLauncher.registerForResult(this) {
             when (it) {
                 is ImageCaptured -> pendingUploadTask?.onReceiveValue(arrayOf(Uri.fromFile(it.file)))
-                CouldNotCapturePermissionDenied -> {
+                is CouldNotCapturePermissionDenied -> {
                     pendingUploadTask?.onReceiveValue(null)
                     activity?.let { activity ->
-                        externalCameraLauncher.showPermissionRationaleDialog(activity)
+                        externalCameraLauncher.showPermissionRationaleDialog(activity, it.input)
                     }
                 }
                 NoImageCaptured -> pendingUploadTask?.onReceiveValue(null)
```

Comment:
Should this also be `is NoImageCaptured` ?

Reply:
Updated, thanks :+1: